jest.setupTestFrameworkScriptFile.js

```diff
@@ -32,6 +32,13 @@ window.guardian = {
 	},
 };
 
+window.fetch = jest.fn().mockImplementation(() =>
```

Comment:
fetch is undefined in jest tests, prebid has now switched to fetch, it isn't actually called in tests but it needs to be there because prebid binds to it [here](https://github.com/prebid/Prebid.js/blob/master/src/ajax.js#L5)